whatlies/transformers/_normalizer.py

```diff
@@ -1,6 +1,6 @@
 from sklearn.preprocessing import normalize
 
-from whatlies.transformers import Transformer
+from ._transformer import Transformer
 from whatlies import EmbeddingSet
 from whatlies.transformers._common import new_embedding_dict
 
```

Comment:
Same as `Noise`, it could have been refactored like this:
```python
    super().__init__(
        FunctionTransformer, f"normalizer_{norm}", func=lambda X: normalize(X, norm=norm, axis=0 if feature else 1), **kwargs
    )
```